go/vt/vtexplain/vtexplain_vttablet.go

```diff
@@ -225,6 +225,7 @@ func (t *explainTablet) ExecuteBatch(ctx context.Context, target *querypb.Target
 	// copy the bindVars into the executor to avoid a data race.
 	for _, query := range queries {
 		bindVariables := sqltypes.CopyBindVariables(query.BindVariables)
+		query.BindVariables = bindVariables
```

Comment:
`query.BindVariables = sqltypes.CopyBindVariables(query.BindVariables)`
Go guarantees this is safe.

Reply:
Updated